opae-libs/cmake/modules/OPAEGit.cmake

```diff
@@ -33,7 +33,7 @@
 
 find_program(OPAE_GIT_EXECUTABLE git)
 
-if(EXISTS ${OPAE_GIT_EXECUTABLE})
+if((EXISTS ${OPAE_GIT_EXECUTABLE}) AND (EXISTS ".git"))
```

Comment:
The .git directory needs to be prefixed with a path, eg ${CMAKE_SOURCE_DIR}/.git

Reply:
fixed in new rev